src/move/econia/sources/market.move

```diff
@@ -2766,13 +2986,19 @@ module econia::market {
                         fill_size, complete_fill, optional_base_coins,
                         quote_coins, fill_size * lot_size,
                         ticks_filled * tick_size);
-                // Get taker events handle.
-                let taker_handle = &mut order_book_ref_mut.taker_events;
-                // Emit corresponding taker event.
-                event::emit_event(taker_handle, TakerEvent{
-                    market_id, side, market_order_id, maker, custodian_id:
-                    maker_custodian_id, size: fill_size, price});
-                // If order on book completely filled:
+                vector::push_back(event_queue_ref_mut, MatchEvent{
```

Comment:
Add space before bracket

Reply:
I agree that the style you are suggesting is more Rust-like and should be used on other codebases moving forward, but per https://github.com/econia-labs/econia#move the Econia Move code is formatted per PEP8, so I am leaning toward consistency within the Econia v4 codebase.

Reply:
See #323

Reply:
Resolving per #323 merge